src/lib/utils/shuffle/shuffle.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it, expect, vi } from "vitest";
+import { shuffle } from "./shuffle";
+
+describe("shuffle function", () => {
```

Comment:
Parmis les tests que j'ai cité dans la review précédente, tu ne test pas :
- vérification de la non-mutabilité de la fonction
- vérifier que l'array mélangé contient chaque élément de l'array précédente (peut-être que si, mais j'aimerais un `it` bien nommé par test d'écrit précédemment)

Reply:
Ok je viens de voir que c'est vérifie plus bas quand même. Malgré ça, split d'avantage les tests, même si le code est souvent explicite, l'objectif et de pouvoir fix une erreur juste en connaissant le message du test qui a fail.